app/grandchallenge/participants/admin.py

```diff
@@ -31,3 +33,10 @@ class RegistrationQuestionAdmin(GuardedModelAdmin):
 admin.site.register(
     RegistrationQuestionGroupObjectPermission, GroupObjectPermissionAdmin
 )
+
+admin.site.register(
```

Comment:
Is `RegistrationQuestionAnswer` missing from the admin?

Reply:
It is, do we need to have access to it in the admin? I don't forsee us ever having to access it.

If we do need it, how is access to the RSEs handled? Is that done live in production?

Reply:
It's good practise to register every model. Access is then defined by group permissions.